app/models/notification.rb

```diff
@@ -54,7 +52,7 @@ def previously_communicated_by?(method)
   end
 
   def preferred_communication_method
-    CountryConfig.current[:name] == "India" ? "missed_visit_whatsapp_reminder" : nil
+    Flipper.enabled?(:whatsapp_appointment_reminders) ? "missed_visit_whatsapp_reminder" : nil
```

Comment:
This was the name of the feature flag before, correct?

Reply:
This is still used in the appointment notification service.

Reply:
I do think we need to separate our flags more carefully. We've already separated out "notifications" into a separate flag, and this should probably be separated into simply a "whatsapp" flag.